iceoryx_examples/icecubetray/README.md

```diff
@@ -0,0 +1,127 @@
+|   Radar App        |           | Privileged Shared     |           |  Cheeky App         |
+|   @perception      |           | Memory Segment        |           |  @notallowed        |
+|                    |  publish  |                       | subscribe |      #     #        |
+|            #       |  -------> | r group: unprivileged |  ------>  |       #   #         |
+|           #        |           | w group: privileged   |           |        # #          |
+|          #         |           |                       |           |        # #          |
+|     #   #          |           |                       |  <------  |       #   #         |
+|      # #           |           |                       |  publish  |      #     #        |
++--------------------+           |                       |           +---------------------+
+                                 +-----------------------+
+                                                 |
+                                                 |          subscribe
+                                                 +---------------------------+
+                                                                             |
+                                                                             |
+                                                                             |
+                                                                            \ /
+                                 +-----------------------+
+                                 |                       |           +---------------------+
+                                 | Infotainment Shared   |           |   Display App       |
+                                 | Memory Segment        |  publish  |   @infotainment     |
+                                 |                       |  <------  |                     |
+                                 | r group: infotainment |           |            #        |
+                                 | w group: infotainment |           |           #         |
+                                 |                       |           |          #          |
+                                 |                       |           |     #   #           |
+                                 |                       |           |      # #            |
+                                 |                       |           +---------------------+
+                                 +-----------------------+
+```
+
+### RouDi and Apps
+
+#### Working setup
+
+RouDi is built with two static shared memory segments _infotainment_ and _privileged_. The access rights of the segments are configured as depicted in the graphic above.
+
+The `roudiConfig` is composed of a memory pool config called `mepooConfig`. When the segement is created, one needs to
+specific the reader group (first string), writer group (second string) as well as the `mepooConfig` (last parameter).
```

Comment:
A short sentence would be good that the assignment of access rights is based on usergroups and not on explicit users.
Maybe an additional hint that all users in the readergroup are allowed to read but not write and the writergroup can do both.